specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimanagement.json

```diff
@@ -530,12 +530,12 @@
       "maxLength": 256,
       "x-ms-parameter-location": "method"
     },
-    "PropertyIdParameter": {
+    "NamedValueIdParameter": {
       "name": "propId",
```

Comment:
>propId [](start = 15, length = 6)

rename to nameValueId

Reply:
once you do that.. you will need to change the examples also.

---
In reply to: [341407943](https://github.com/Azure/azure-rest-api-specs/pull/7683#discussion_r341407943) [](ancestors = 341407943)

Reply:
Done